riva/client/argparse_utils.py

```diff
@@ -85,6 +85,24 @@ def add_asr_config_argparse_parameters(
         type=float,
         help="Threshold value for likelihood of blanks before detecting end of utterance",
     )
+    parser.add_argument(
+        "--source-language",
+        default="",
+        type=str,
+        help="Language of the audio file",
+    )
+    parser.add_argument(
+        "--task",
+        default="transcribe",
+        type=str,
+        help="Task for the model (transcribe/translate)",
+    )
+    parser.add_argument(
+        "--target-language",
+        default="",
+        type=str,
+        help="Target language for translation",
+    )
```

Comment:
we should support these command line params through the generic --metadata field we have in the client. Adding dedicated translation specific params (source/target language, task) can be confusing for users in the ASR client.

Reply:
Updated to use custom_configuration